custom_dc/index.md

```diff
@@ -95,3 +95,13 @@ The cost of running a site on Google Cloud Platform depends on the size of your
 1. Build a custom image, upload it to the Google Cloud Artifact Registry and create a Cloud Run service to run the site. See [Deploy services to Google Cloud](deploy_cloud.md)
 1. Launch and productionize your site for external traffic. See [Launch your Data Commons](/custom_dc/launch_cloud.html).
 1. For future updates and launches, continue to make UI and data changes locally, before deploying the changes to GCP.
+
+## Send feedback {#feedback}
+
+We use [Google Issue Tracker](https://issuetracker.google.com){: target="_blank"} to track bugs and feature requests. All tickets are publicly viewable.
+
+- To make a feature request, please see if an [existing request](https://issuetracker.google.com/issues?q=componentid:1659535%2B%20type:feature_request){: target="_blank"} has already been filed. If yes, upvote (click the **+1** button ) and [subscribe](https://developers.google.com/issue-tracker/guides/subscribe){: target="_blank"} to it. If not, open a [new feature request](https://issuetracker.google.com/issues/new?component=1659535&template=2053231){: target="_blank"}.
```

Comment:
Consider mentioning the upvote collectively for both types of issues.

Reply:
Done.